doc/how_to_add_a_new_source_file_to_specfem_2D.txt

```diff
@@ -0,0 +1,50 @@
+STEPS TO ADD A SOURCE FILE TO SPECFEM2D
+=======================================
+
+1. Create your source file, for example my_sexy_subroutine.f90 :
+
+      ! Comment here to explain why your subroutine is sexy
+      subroutine my_sexy_subroutine(nothing_funnier_than_that_int)
+
+        use specfem_par, only: nspec,nglob,ibool,coord
+
+        implicit none
+        include "constants.h"
```

Comment:
`specfem_par` includes all constants. There should be no reason to include the header as well.